apps/cookbook/src/app/showcase/page-showcase/page-showcase.component.html

```diff
@@ -56,11 +56,15 @@ <h3>Life-cycle hooks</h3>
   <p>
     The <code>kirby-router-outlet</code> stacks the pages/components when it navigates. This means
```

Comment:
Revert the changes to this file as they are not related to the issue at hand 😃

Reply:
Yes sir should be fixed